### ISSUE TITLE
"Start with windows" feature should be renamed to "On user logon"

### ISSUE THREAD

**Issue by 2korzhik** [Rem0o/FanControl.Releases#1809]
Fan speed is not beeing managed by FanControl app until user log on event occurs. 
I have multiuser windows setup with "**windows user auto log on**" feature disabled. So my fans are at 60% rpm (motherboard bios limitation, cannot make lower) until i login with any user. 

**Relevant hardware specs and setup**
win10, asus maximus gene V motherboard, latest bios

**Comment by Rem0o** [comment]
If you start the app without any user, you won't get a UI, and will be run through the SYSTEM user. As you try and open the app from a real user, you create a new process, which then conflicts with the SYSTEM one, hence why it will close that one down before opening the new one, hence the message.

**Comment by 2korzhik** [comment]
Yes i know, so what? What can be done to have both: the app starts with windows (before user logon) and can be managed with gui on demand by any user?
